case_studies/CARD/Bug-Drug Code.R

```diff
@@ -0,0 +1,257 @@
+# config.R
+url <- "https://card.mcmaster.ca/download/0/broadstreet-v3.3.0.tar.bz2"
+destfile <- "broadstreet-v3.3.0.tar.bz2"
+
+# Download the file
+download.file(url, destfile)
+
+#Extract the file
+if (!require("R.utils")) {
+  install.packages("R.utils")
+  library(R.utils)
+}
+
+
+# Extract the tar file
+untar("broadstreet-v3.3.0.tar.bz2", exdir = "CARD_data")
+
+
+# Map CARD Short Name
+
+# Parse the required files using readr::read_delim
+aro_index <- read_delim("CARD_data/aro_index.tsv", delim = "\t", col_names = TRUE)
+antibiotics_data <- read_delim("CARD_data/shortname_antibiotics.tsv", delim = "\t", col_names = TRUE)
+pathogens_data <- read_delim("CARD_data/shortname_pathogens.tsv", delim = "\t", col_names = TRUE)
+
+
+
+# Extract pathogen, gene, drug, and include Protein.Accession from 'CARD Short Name'
+extract_card_info <- function(card_short_name, drug_class, `Protein Accession`, `DNA Accession`) {
```

Comment:
rename all colnames with spaces and special characters to now include only `_`. Also avoid multiple cases.

@AbhirupaGhosh @charmvang @awasyn @epbrenner @the-mayer -- using camelCase for colnames or snake_case (without caps)?